demo-resources/.htaccess

```diff
@@ -5,41 +5,62 @@ Options -MultiViews
-RewriteCond %{HTTP:Accept} text/turtle [NC]
-RewriteRule ^demo-classifications# https://jimjyang.github.io/showroom/xkos-ap-no/files/demo-classifications.ttl# [R=302,L]
+RewriteRule ^(demo-classifications)$ https://jimjyang.github.io/showroom/xkos-ap-no/files/$1-en [R=303,L] 
 
 #Rewrite rules for w3id.org/demo-resources/demo-concepts
+# text/turtle
+RewriteCond %{HTTP:Accept} text/turtle [NC]
+RewriteRule ^(demo-concepts)$ https://jimjyang.github.io/showroom/skos-ap-no/files/$1.ttl [R=302,L]
 # Norwegian Bokmål
 RewriteCond %{HTTP:Accept-Language} nb [NC]
-RewriteRule ^demo-concepts# https://jimjyang.github.io/showroom/skos-ap-no/files/demo-concepts-nb#  [R=302,L]
+RewriteRule ^(demo-concepts)$ https://jimjyang.github.io/showroom/skos-ap-no/files/$1-nb  [R=302,L]
+# Norwegian Nynorsk
+RewriteCond %{HTTP:Accept-Language} nn [NC]
+RewriteRule ^(demo-concepts)$ https://jimjyang.github.io/showroom/skos-ap-no/files/$1-nb  [R=303,L]
+# Norwegian Norsk
+RewriteCond %{HTTP:Accept-Language} no [NC]
+RewriteRule ^(demo-concepts)$ https://jimjyang.github.io/showroom/skos-ap-no/files/$1-nb  [R=303,L]
 # English
 RewriteCond %{HTTP:Accept-Language} en [NC]
-RewriteRule ^demo-concepts# https://jimjyang.github.io/showroom/skos-ap-no/files/demo-concepts-en#  [R=302,L]
+RewriteRule ^(demo-concepts)$ https://jimjyang.github.io/showroom/skos-ap-no/files/$1-en  [R=302,L]
 # English as default
-RewriteRule ^demo-concepts# https://jimjyang.github.io/showroom/skos-ap-no/files/demo-concepts-en#  [R=303,L]
-RewriteCond %{HTTP:Accept} text/turtle [NC]
-RewriteRule ^demo-concepts# https://jimjyang.github.io/showroom/skos-ap-no/files/demo-concepts# [R=302,L]
+RewriteRule ^(demo-concepts)$ https://jimjyang.github.io/showroom/skos-ap-no/files/$1-en [R=303,L]
 
 #Rewrite rules for w3id.org/demo-resources/demo-models
+# text/tutrle
+RewriteCond %{HTTP:Accept} text/turtle [NC]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1.ttl [R=302,L]
 # Norwegian Bokmål
 RewriteCond %{HTTP:Accept-Language} nb [NC]
-RewriteRule ^demo-models# https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-nb#  [R=302,L]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-nb  [R=302,L]
+# Norwegian Nynorsk
+RewriteCond %{HTTP:Accept-Language} nn [NC]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-nb  [R=303,L]
+# Norwegian Norsk
+RewriteCond %{HTTP:Accept-Language} no [NC]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-nb  [R=303,L]
 # English
 RewriteCond %{HTTP:Accept-Language} en [NC]
-RewriteRule ^demo-models# https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-en#  [R=302,L]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-en  [R=302,L]
```

Comment:
In https://github.com/jimjyang/showroom/tree/main/modelldcat-ap-no/files you have no files ended in `en`, so this redirection will fail. Maybe you mean https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-en.html#$1 ?

Reply:
@dgarijo Thanks for your comment! 

https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-en does work. Or, what did you mean?

Reply:
Mhh, my bad, it looks like I tested it out the wrong file. Apologies!

Reply:
Thanks!